src/main/webapp/WEB-INF/web.xml

```diff
@@ -76,6 +76,7 @@
     <url-pattern>/register</url-pattern>
   </servlet-mapping>
 
+  
```

Comment:
This blank line can be reverted (that way there won't be a diff in the history of this file).